### ISSUE TITLE
7.0.1-nvidia2204 No such filter: 'drawtext'

### ISSUE THREAD

**Issue by lemonyxk** [jrottenberg/ffmpeg#403]
```
root@media-server-1:~# ffmpeg -hwaccel_device 0 -hwaccel cuvid -hwaccel_output_format cuda -c:v h264_cuvid -i /app/media-server-storage/603530197806096a/603530197806096a -fps_mode cfr -threads 0 -filter_complex "[0:v]scale_npp=1280:720:format=yuv420p,hwdownload,format=yuv420p,pad=1280:720:0:0:black,drawtext=text='hello':fontsize=24:fontcolor=white@0.50:'x=if(gte(mod(t,30)*100,20),if(lte(mod(t,30)*100,W-tw-20),mod(t,30)*100,-tw),-tw):y=20:enable=mod(t,30)'" -c:v h264_nvenc -c:a copy -b:v 356206 -b:a 64994 -maxrate 421200 -minrate 421200 -bufsize 842400 -g 50 -keyint_min 50 -sc_threshold 0 -muxpreload 0 -muxdelay 0 -force_key_frames 00:00:00.000 -r 25 -tag:v hvc1 -hls_playlist_type vod -f hls -hls_segment_type fmp4 -hls_time 4 -hls_flags single_file -hls_list_size 0 -hls_segment_filename origin.mp4 origin.m3u8 -y -progress - -loglevel 24
[AVFilterGraph @ 0x557e455b7880] No such filter: 'drawtext'
Failed to set value `'[0:v]scale_npp=1280:720:format=yuv420p,hwdownload,format=yuv420p,pad=1280:720:0:0:black,drawtext=text='hello':fontsize=24:fontcolor=white@0.50:'x=if(gte(mod(t,30)*100,20),if(lte(mod(t,30)*100,W-tw-20),mod(t,30)*100,-tw),-tw):y=20:enable=mod(t,30)''` for option 'filter_complex': Filter not found
Error parsing global options: Filter not found
root@media-server-1:~# ffmpeg -v
ffmpeg version 7.0.1 Copyright (c) 2000-2024 the FFmpeg developers
  built with gcc 11 (Ubuntu 11.4.0-1ubuntu1~22.04)
  configuration: --disable-debug --disable-doc --disable-ffplay --enable-cuda --enable-cuvid --enable-fontconfig --enable-gpl --enable-libaom --enable-libaribb24 --enable-libass --enable-libbluray --enable-libfdk_aac --enable-libfreetype --enable-libkvazaar --enable-libmp3lame --enable-libnpp --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libsrt --enable-libtheora --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --enable-libzimg --enable-libzmq --enable-nonfree --enable-nvenc --enable-openssl --enable-postproc --enable-shared --enable-small --enable-version3 --extra-cflags='-I/opt/ffmpeg/include -I/opt/ffmpeg/include/ffnvcodec -I/usr/local/cuda/include/' --extra-ldflags='-L/opt/ffmpeg/lib -L/usr/local/cuda/lib64 -L/usr/local/cuda/lib32/' --extra-libs=-ldl --extra-libs=-lpthread --prefix=/opt/ffmpeg
  libavutil      59.  8.100 / 59.  8.100
  libavcodec     61.  3.100 / 61.  3.100
  libavformat    61.  1.100 / 61.  1.100
  libavdevice    61.  1.100 / 61.  1.100
  libavfilter    10.  1.100 / 10.  1.100
  libswscale      8.  1.100 /  8.  1.100
  libswresample   5.  1.100 /  5.  1.100
  libpostproc    58.  1.100 / 58.  1.100
Missing argument for option 'v'.
Error splitting the argument list: Invalid argument

it works well on ffmpeg:5.1.2-nvidia2004
```

**Comment by maobole** [comment]
7.0.2-alpine313 output the same error

work well on 5.1.6-alpine313

**Comment by Gwemox** [comment]
Same error in 6.1.

Same error in 6.1, I think '--enable-libharfbuzz' is missing as indicated in the documentation.

**Comment by jsheffie** [comment]
@Gwemox there are 7.1 builds on [hub.docker](https://hub.docker.com/r/jrottenberg/ffmpeg/tags?name=7.1) now. 
Specifically `docker pull jrottenberg/ffmpeg:7.1-nvidia`
![Screenshot 2024-10-29 at 1 18 48 PM](https://github.com/user-attachments/assets/41d61602-fb3d-4025-823d-8067c3ecf2f3)

**Comment by Gwemox** [comment]
> @Gwemox there are 7.1 builds on [hub.docker](https://hub.docker.com/r/jrottenberg/ffmpeg/tags?name=7.1) now. Specifically `docker pull jrottenberg/ffmpeg:7.1-nvidia`
> 
> ![Screenshot 2024-10-29 at 1 21 30 PM](https://private-user-images.githubusercontent.com/890028/381258880-a1de1648-33e5-4e54-afa6-653b973011ec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzMDQzMDAsIm5iZiI6MTczMDMwNDAwMCwicGF0aCI6Ii84OTAwMjgvMzgxMjU4ODgwLWExZGUxNjQ4LTMzZTUtNGU1NC1hZmE2LTY1M2I5NzMwMTFlYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAzMFQxNjAwMDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lZTVjZGE1MzBkZjYwOTlhNzYwNjI0MWYzMjcwMmI1NWJjOWJlMDdkZGRmM2I5YmFhZGNhYTc5YjcwZTFjOTZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.L1GzOi8cIdVKg4N_OBENxCrn8_NqQbJN1VDW-UU4CKI)

Thank you for your work; I’ve responded in [the thread of your PR](https://github.com/jrottenberg/ffmpeg/pull/408#issuecomment-2447624435).

**Comment by jsheffie** [comment]
this is fixed in #413, recommend closing this issue
- #413
